### ISSUE TITLE
[React Native Gallery -> VirtualizedList]: Unable to select/unselect multiple list items present under "A VirtualizedList with multiple selection support" group in one go using keyboard.

### ISSUE THREAD

**Issue by YajurG** [microsoft/react-native-gallery#366]
**Repro-Steps:**

- Launch the React Native Gallery app.
- Navigate to VirtualizedList’ button and invoke it.
- Navigate to 'Selection Support' combo box present under "A VirtualizedList with multiple selection support" group and expand it.
- Navigate to 'Multiple' and invoke it.
- Now navigate to list items present under "A VirtualizedList with multiple selection support" and select/unselect and observe focus.

**Actual Result:**

Unable to select/unselect multiple list items present under "A VirtualizedList with multiple selection support" group in one go using keyboard. When user select/unselect any one list item keyboard focus is landing on 'Selection Support' combo box, which is incorrect behaviour.

**Expected Result:**
User should be able to select/unselect multiple list items in one go. When user select/unselect any list item keyboard focus should not redirect to 'Selection Support' combo box. Focus is stay on the selected/unselected list item.

**User Impact:**
Users with motor disabilities who rely on keyboard will face difficulties if keyboard focus redirects to other interactive elements i.e., on 'Selection Support' combo box.

Internal Bug Link: https://dev.azure.com/microsoft/OS/_workitems/edit/47844267
